iothub/device/src/Transport/AmqpIoT/AmqpIoTCbsConstants.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Amqp;
+
+namespace Microsoft.Azure.Devices.Client.Transport.AmqpIoT
+{
+    internal class AmqpIoTCbsConstants
+    {
+        internal const string IotHubSasTokenType = CbsConstants.IotHubSasTokenType;
```

Comment:
We can probably merge this class with something else to avoid creating a new type. That will save a few bytes in the binary.

Reply:
Moved to general contants file

---
In reply to: [292146366](https://github.com/Azure/azure-iot-sdk-csharp/pull/947#discussion_r292146366) [](ancestors = 292146366)